style.css

```diff
@@ -0,0 +1,193 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  font:
+    normal "Lucida Grande",
+    "Lucida Sans",
+    "Lucida Sans Regular",
+    "Lucida Sans Unicode",
+    Geneva,
+    Verdana,
+    sans-serif;
+}
+
+body a {
+  text-decoration: none;
+}
+
+nav a {
+  font-size: 1.5em;
+}
+
+footer a {
+  color: #08c;
+}
+
+body p a {
+  color: #08c;
+}
+
+/* -- Nav -- */
+
+header {
+  color: white;
+  width: 100%;
+  height: 100vh;
+  background: url("/img/main-img.jpg");
+  background-position: right bottom;
+  background-size: cover;
+}
+
+nav {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-weight: 100;
+  font-family: "Lato", sans-serif;
+}
+
+nav ul {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 5em;
+  list-style: none;
+  font-weight: bolder;
+  font-size: 15px;
+  border: 0.8px solid #888;
+  border-radius: 4px;
+  background:
+    -webkit-gradient(
+      linear,
+      0 0,
+      0 100%,
+      from(rgba(0, 0, 0, 0.2)),
+      color-stop(0.05, rgba(0, 0, 0, 0)),
+      color-stop(0.97, rgba(0, 0, 0, 0)),
+      to(rgba(0, 0, 0, 0.45))
+    ),
+    -webkit-gradient(linear, 0 0, 100% 0, from(rgba(0, 0, 0, 0.2)), color-stop(0.002, rgba(0, 0, 0, 0)), color-stop(0.998, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, 0.2))),
+    no-repeat 50% 50%,
+    -webkit-gradient(linear, 0 100%, 0 0, from(#666), color-stop(0.5, #5e5e5e), color-stop(0.51, #707070), to(#808080));
+}
+
+footer li {
+  display: inline;
+  padding-left: 0.6em;
+  margin-left: 0.6em;
+  border-left: 1px solid #ccc;
+}
+
+.nav-menu li {
+  margin: -3px 0;
```

Comment:
Maybe you can remove this negative margin to fix the issue with the dividers. Just an idea, not sure if that's the issue or if removing that will mess up something else.